### DISCUSSION TITLE
SNES BOM Question

### DISCUSSION THREAD

**Discussion by joeybaumgartner** [pmgducati/Blue-Retro-AIO-Units#48] [Q&A]
I was looking at the Excel and PDF files for the BOM. For the SNES extension, it just lists 1 extension required. The main BlueRetro project lists 2, and I believe you need 7 conductors for it act as a multi-tap. Does the SNES PCB for this project list only work as a single player, or would another PCB be required to make this work as a multi-tap device?

**Comment by amaiorano** [comment]
This is a mistake, and should say 2.

Note that if you want multitap, though, SNES extension cables will not work as they're missing one wire. The [official build instructions from Darthcloud say as much](https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#bill-of-materials-11):

```
SNES controller plug (x2) (For multitap support 7 conductor are required, get a 3rd SNES extension for pin donation and use cable from PSX/Genesis/Saturn for the extra wires)
```

~~Another option is to use cables from real SNES controllers -- or SFC ones, which are shorter and cheaper. Also, to support multitap, you need to solder a third level shifter (74AHCT125N) onto the PCB.~~ EDIT: scratch that, I'm wrong, the OG controllers do not have an extra 7th wire in them. The extra wire is only found in the four score devices themselves. So Darthcloud's instructions still stand.

**Comment by pmgducati** [comment]
The link in the BOM is for a 2 pack of extension cables, the Qty is not a mistake.  I updated the link to the console5 extension that is sold in single qty to remove future confusion and to be consistent with the other systems.  I also made a call out referencing DarthClouds instructions.
